### PR TITLE
RichCommand: Show Properties

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Description

Add a RichCommand Show Properties

### Motivation and Context

- Related #11481
- Add a RichCommand Show Properties
- Add XAML comments in InnerNavigationToolbar.xaml

### PR Checklist
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
- [x] Did you build the app and test your changes?
- [x] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### Screenshots

_None_

### CODE REVIEW

**Comment by ferrariofilippo** [comment]
What about a keyboard shortcut? Maybe `Ctrl + P` (Properties) or `Ctrl + I` (Info)?

**Comment by yaira2** [comment]
> What about a keyboard shortcut? Maybe Ctrl + P (Properties) or Ctrl + I (Info)?

We can leave without and let the user assign their own.

File: src/Files.App/Actions/Toggle/TogglePreviewPaneAction.cs
Comment (yaira2) [inline src/Files.App/Actions/Toggle/TogglePreviewPaneAction.cs:1]: Can you revert this change?

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml [hunk 1]
```diff
@@ -180,13 +180,12 @@
...
-					IsEnabled="{x:Bind converters:MultiBooleanConverter.AndConvert(ViewModel.CanViewProperties, ViewModel.InstanceViewModel.IsPageTypeNotHome), Mode=OneWay, FallbackValue=False}"
-					Label="{helpers:ResourceString Name=Properties}"
+					Command="{x:Bind Commands.ShowProperties, Mode=OneWay}"
+					IsEnabled="{x:Bind Commands.ShowProperties.IsExecutable, Mode=OneWay, FallbackValue=False}"
+					Label="{x:Bind Commands.ShowProperties.Label, Mode=OneWay}"
 					LabelPosition="Collapsed"
-					ToolTipService.ToolTip="{helpers:ResourceString Name=Properties}">
-
-					<local:OpacityIcon Style="{StaticResource ColorIconProperties}" />
+					ToolTipService.ToolTip="{x:Bind Commands.ShowProperties.Label, Mode=OneWay}">
```
Comment (yaira2) [inline src/Files.App/UserControls/InnerNavigationToolbar.xaml:187]:
```suggestion
					ToolTipService.ToolTip="{x:Bind Commands.ShowProperties.LabelWithHotKey, Mode=OneWay}">
```

**Comment by yaira2** [comment]
> Add XAML comments in InnerNavigationToolbar.xaml

Can you open a separate PR for this? It makes it easier to refer back to the change (it also separates the changes in the changelog).

**Comment by 0x5bfa** [comment]
It's annoying Tests frequently fails:(

File: src/Files.App/Helpers/ContextFlyoutItemHelper.cs [hunk 2]
```diff
@@ -387,20 +387,11 @@ private static bool Check(ContextMenuFlyoutItemViewModel item, CurrentInstanceVi
...
-					{
-						OpacityIconStyle = "ColorIconProperties",
-					},
-					Command = commandsViewModel.ShowPropertiesCommand,
-					ShowInRecycleBin = true,
-					ShowInSearchPage = true,
-					ShowInFtpPage = true,
-					ShowInZipPage = true,
-				},
+					IsVisible = commands.ShowProperties.IsExecutable && itemsSelected,
```
Comment (yaira2) [inline src/Files.App/Helpers/ContextFlyoutItemHelper.cs:393]: Shouldn't `itemsSelected` be part of `IsExecutable`?

Comment (0x5bfa) [reply src/Files.App/Helpers/ContextFlyoutItemHelper.cs:393]: Of course:)

**Review by yaira2 — DISMISSED** [review]
LGTM

**Comment by yaira2** [comment]
@0x5bfa can you fix the merge conflict and look into why it's failing the automation tests?

**Comment by 0x5bfa** [comment]
I will try again.
